spec/features/accordion_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+describe "Accordion", js: true, type: :view do
+  scenario "There is a page with the accordion rendered" do
+    given_i_visit_a_page_with_the_accordion_component
+    then_the_accordion_loads
+    then_the_accordion__javascript_loads
+    when_i_click_the_first_accordion_section
+    then_the_accordion_opens
+    when_i_click_the_first_accordion_section
+    then_the_accordion_has_data_attributes
+  end
+
+  def given_i_visit_a_page_with_the_accordion_component
+    visit "/accordionexample"
+  end
+
+  def then_the_accordion_loads
+    expect(page).to have_css(".gem-c-accordion", visible: true)
```

Comment:
Do you think it's worth checking the accordion hasn't got any sections open when it's loaded?

```ruby
expect(page).not_to have_content("This is the content for Writing well for the web.")
```